src/config.rs

```diff
@@ -1242,4 +1298,15 @@ mod tests {
 
         assert_eq!(final_config.get_permissions().len(), 1)
     }
+
+    /// Test to check the default reports to be generated
+    #[test]
+    fn it_generates_html_but_not_json_by_default() {
+        let mut final_config = Config::default();
+        final_config.html = false;
+        final_config.json = false;
+
+        assert!(final_config.has_to_generate_html() == true);
```

Comment:
No need to compare to true or false, simply `assert!(final_config.has_to_generate_html());` or `assert!(!final_config.has_to_generate_json())`.